app/models/project.js

```diff
@@ -54,6 +58,12 @@ export default class ProjectModel extends Model {
     return dcpPlannedstartdate || null;
   }
 
+  @computed('milestones')
+  get tabSpecificMilestones() {
+    const tabSpecificMilestones = this.milestones.filter(milestone => this.milestoneConstants.milestoneListByTabLookup[this.tab].includes(milestone.dcpMilestone));
+    return tabSpecificMilestones;
```

Comment:
Style nit but I would just return this without creating a new variable name:

```
return this.milestones.filter(milestone => this.milestoneConstants.milestoneListByTabLookup[this.tab].includes(milestone.dcpMilestone));
```

No?

Reply:
If it's because you want to console log the variable, you can just reference it in a test with whatever value you're looking for

Reply:
oh woops, yes I was exactly doing it for the console logging and forgot to change it back 😣